Application/TuitionPartnerShortlistStorage/Implementations/CookieBasedTuitionPartnerShortlistStorage.cs

```diff
@@ -0,0 +1,140 @@
+namespace Application.TuitionPartnerShortlistStorage.Implementations;
```

Comment:
I think we should move this into the UI project so that we can use the `CookieHeaderValue` stuff.